test/regressions/TestViewer.js

```diff
@@ -2,7 +2,8 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { useFakeTimers } from 'sinon';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/styles';
+import { createTheme } from '@material-ui/data-grid';
```

Comment:
@DanailH is it fine to do this import from here?

Reply:
Yes it's ok